Docker/validate_sc2.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+"""Validate SC2."""
+
+import argparse
+import json
+
+import pandas as pd
+
+
+def main(submission, entity_type, results="results.json"):
+    """Validate submission and write results to JSON.
+
+    Args:
+        submission: input file
+        entity: Synapse entity type
+        results: output file
+    """
+
+    invalid_reasons = []
+
+    if not submission.endswith(".csv"):
```

Comment:
A better way to do this is to try except the read csv .  Basically if someone submits a Excel file, but names it a .csv, this would fail.